pages/index.vue

```diff
@@ -11,3 +51,46 @@ import { Component, Vue } from "nuxt-property-decorator"
 @Component
 export default class Index extends Vue {}
 </script>
+<style type="text/css">
```

Comment:
this is the default and I don't think has to be included. Not bad to be verbose though.